packages/typespec-ts/test/integration/encodeBytes.spec.ts

```diff
@@ -186,4 +187,132 @@ describe("EncodeDatetimeClient Rest Client", () => {
     });
 
   });
-});
\ No newline at end of file
+
+  describe("request body", () => {
+    const pngFile = readFileSync(resolve("../../packages/typespec-ts/temp/assets/image.png"));
```

Comment:
Could you help me understand why the case can be passed in ci? I didn't find we have a image.png file in temp folder.

Reply:
I notice these cases are skipped?

Reply:
the test cases are skipped but we can read the file because I have changed the string to copy the whole cadl-ranch-specs folder not just http folder https://github.com/Azure/autorest.typescript/pull/2032/files#diff-717e8cb22e747bf40ac8bb73b4111d6e4478be1e4e92f821531028d4e136e766R23 here.